src/pages/MeetingPage/MeetingPage.tsx

```diff
@@ -36,42 +81,60 @@ const MeetingPage = () => {
     } catch (error) {
       utm = {}
     }
-    fetch(process.env.REACT_APP_API_BASE_ROOT + '/sys/create-lead', {
-      method: 'post',
-      body: JSON.stringify({
-        phone,
-        email,
-        name,
-        managerUsername,
-        taskTitle: `專屬預約諮詢:${timeslots.join('/')}`,
-        categoryNames: fields,
-        properties: [
-          { name: '介紹人', value: referal },
-          { name: '名單分級', value: 'SSR' },
-          { name: '來源網址', value: window.location.href },
-          { name: '聯盟來源', value: utm.utm_source || '' },
-          { name: '聯盟會員編號', value: utm.utm_id || '' },
-          { name: '聯盟成交編號', value: utm.utm_term || '' },
-        ],
-      }),
-      credentials: 'include',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-    })
-      .then(res => res.json())
-      .then(({ code, message }: { code: string; message: string }) => {
-        if (code === 'SUCCESS') {
+
+    const postAdProperty = [
+      { name: '介紹人', value: referal },
+      { name: '名單分級', value: 'SSR' },
+      { name: '來源網址', value: window.location.href },
+      { name: '聯盟來源', value: utm.utm_source || '' },
+      { name: '聯盟會員編號', value: utm.utm_id || '' },
+      { name: '聯盟成交編號', value: utm.utm_term || '' },
+    ]
+
+    if (isInsertingAdProperty) {
+      postAdProperty.push({ name: '廣告素材', value: 'inbound_英鎊' })
+    }
+
+    if (currentMemberId !== null && isInsertingAdProperty) {
+      updateMemberCreated()
+        .then(() => updateMemberProperties())
+        .finally(() => {
           Cookies.remove('utm')
           alert('已成功預約專屬諮詢！')
-        } else {
-          alert(`發生錯誤，請聯繫 contact@xuemi.co。錯誤訊息：${message}`)
-        }
-      })
-      .finally(() => {
-        setIsSubmitting(false)
-        window.location.reload()
+          setIsSubmitting(false)
+          window.location.reload()
+        })
```

Comment:
if current member is exist, will missing `insrt member task`、`update member phone`、`update membr categories`、`update member`、`update tracking log ( utm )`

call create-lead api
reffer:  https://github.com/urfit-tech/lodestar-app-backend/blob/develop/src/controllers/sys.controller.ts:L1233